src/components/organisms/TriggersFormItems/ActionFormItems.tsx

```diff
@@ -0,0 +1,64 @@
+import {useState} from 'react';
+import {useFirstMountState} from 'react-use';
+
+import {Form, Select, Space} from 'antd';
+
+import {LabelsSelect} from '@molecules';
+
+import {useStore} from '@store';
+
+import {required} from '@utils/form';
+
+import ResourceTriggerSelect from './ResourceTriggerSelect';
+import TriggerSelectorSwitcher from './TriggerSelectorSwitcher';
+
+const ActionFormItems = () => {
+  const {triggersKeyMap} = useStore(state => ({
+    triggersKeyMap: state.triggersKeyMap!,
+  }));
+
+  const [switcherValue, setSwitcherValue] = useState<string | number>('label');
```

Comment:
Based on the code, it's rather always `string`, or even more `'name' | 'label'`. Could be extracted to literal anyway.